src/pages/[slug]/[ie]/index.tsx

```diff
@@ -368,12 +383,16 @@ const ObjectDetailPage: NextPage = () => {
 					title={mediaInfo?.maintainerName ?? ''}
 					backLink={backLink}
 					showAccessEndDate={
-						accessEndDate
-							? t(
-									'pages/leeszaal/reading-room-slug/object-id/index___toegang-tot-access-end-date',
-									{ accessEndDate }
-							  )
-							: ''
+						accessEndDate || accessEndDateMobile
+							? isMobile
+								? t(
+										'pages/leeszaal/reading-room-slug/object-id/index___toegang-tot-access-end-date',
+										{ accessEndDate }
+								  )
+								: t('pages/slug/ie/index___tot-access-end-date-mobile', {
+										accessEndDateMobile,
+								  })
+							: undefined
```

Comment:
Ok for now maar persoonlijk hou ik zoveel mogelijk `isMobile`-achtige vars uit het framework, om dit aan te pakken in de toekomst zou ik `showAccessEndDate` omzetten naar `accessEnd?: ReactNode`, twee elementen met aparte inhoud er in pompen met elk een klasse en een bijhorend breakpoint.